rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -40,6 +40,11 @@
 # TODO: Perhaps have a saved version of that global DB for the tests and query it too?
 ASSETS_IN_V2_GLOBALDB = 3095
 YEARN_V1_ASSETS_IN_V3 = 32
+UNIQUE_CACHE_KEYS: set[GeneralCacheType] = {
```

Comment:
These are already defined elsewhere. You shouljd just import them

Reply:
You used a different one in the db upgrade which is fine, but then youshould use here the one you put in the DB upgrade.

Reply:
alright!

Can i change it then?